core/dbt/parser/manifest.py

```diff
@@ -1425,7 +1444,7 @@ def _process_semantic_information_for_node(
 
             # Create a list of models that have relationships pointed at them that
             # have not been set to is_public. Use this list to raise CompilationException
-            if to_model.is_public == False:
+            if not to_model.is_public:
```

Comment:
is this a code-smell change? is there one that our style guide recommends?

Reply:
same as above -- the `flake8` linter flagged this, and I could interpret it (unlike `mypy`) so i figured why not